extractjava.py

```diff
@@ -27,6 +29,8 @@
             for e in expr:
                 try:
                     e.decode("utf8")
+                    if len(e) > 1000:
```

Comment:
What does 1000 refer to?

Reply:
This refers to the total character length of the expression.

Reply:
Can we document that in some way? I wasn't aware there was a limit or that this was what it referred to!

Reply:
I only did this to cut down the runtime of the experiment, since there are some crazy long PHP expressions in the corpus, which are not interesting enought to warrant leaving them in.

Reply:
That's fine -- but we have to document it (including in the paper).